src/test/java/uk/gov/hmcts/reform/civil/handler/callback/user/EvidenceUploadHandlerTest.java

```diff
@@ -1,76 +1,246 @@
     JacksonAutoConfiguration.class
 })
-public class EvidenceUploadHandlerTest extends BaseCallbackHandlerTest {
+class EvidenceUploadHandlerTest extends BaseCallbackHandlerTest {
 
     @Autowired
     private EvidenceUploadHandler handler;
 
+    @MockBean
+    private Time time;
+
+    @Autowired
+    private final ObjectMapper mapper = new ObjectMapper();
+
+    private final UploadEvidenceExpert uploadEvidenceDate = new UploadEvidenceExpert();
+    private final UploadEvidenceWitness uploadEvidenceDate2 = new UploadEvidenceWitness();
+
+    private static final String PAGE_ID = "validateValues";
+
+    @BeforeEach
+    void setup() {
+        given(time.now()).willReturn(LocalDateTime.now());
+    }
+
     @Test
     void givenAboutToStartThenReturnsAboutToStartOrSubmitCallbackResponse() {
+        // Given
         CaseData caseData = CaseDataBuilder.builder().build();
         CallbackParams params = callbackParamsOf(caseData, ABOUT_TO_START);
 
+        // When
         CallbackResponse response = handler.handle(params);
 
+        // Then
         assertThat(response).isInstanceOf(AboutToStartOrSubmitCallbackResponse.class);
     }
 
     @Test
-    void givenAboutToSubmitThenReturnsAboutToStartOrSubmitCallbackResponse() {
-        CaseData caseData = CaseDataBuilder.builder().build();
-        CallbackParams params = callbackParamsOf(caseData, ABOUT_TO_SUBMIT);
+    void shouldSucceed_whenExpertOption1UploadDatePast() {
+        // Given
+        List<Element<UploadEvidenceExpert>> date = newArrayList();
+        date.add(0, element(uploadEvidenceDate.toBuilder()
+                                .expertOption1UploadDate(time.now().toLocalDate().minusWeeks(1)).build()));
+
+        CaseData caseData = CaseDataBuilder.builder().atStateNotificationAcknowledged().build().toBuilder()
+            .documentUploadExpert1(date)
+            .build();
+        CallbackParams params = callbackParamsOf(caseData, MID, PAGE_ID);
+
+        // When
+        var response = (AboutToStartOrSubmitCallbackResponse) handler.handle(params);
+
+        // Then
+        assertThat(response.getErrors()).isEmpty();
+    }
 
-        CallbackResponse response = handler.handle(params);
+    @Test
+    void shouldNotReturnError_whenExpertOption1UploadDatePresent() {
+        // Given
+        List<Element<UploadEvidenceExpert>> date = newArrayList();
+        date.add(0, element(uploadEvidenceDate.toBuilder()
+                                            .expertOption1UploadDate(time.now().toLocalDate()).build()));
+
+        CaseData caseData = CaseDataBuilder.builder().atStateNotificationAcknowledged().build().toBuilder()
+            .documentUploadExpert1(date)
+            .build();
+        CallbackParams params = callbackParamsOf(caseData, MID, PAGE_ID);
+
+        // When
+        var response = (AboutToStartOrSubmitCallbackResponse) handler.handle(params);
+
+        // Then
+        assertThat(response.getErrors()).isEmpty();
+    }
```

Comment:
Using the approach used in `shouldReturnError_whenExpertOptionUploadDateFuture`, make this test run against both `expertOption1UploadDate` and `expertOption2UploadDate`.